samigo/samigo-app/src/webapp/jsf/delivery/item/deliverShortAnswer.jsp

```diff
@@ -30,8 +30,13 @@ should be included in file importing DeliveryMessages
 <%@ include file="/jsf/delivery/item/attachment.jsp" %>
 
 <h:panelGrid rendered="#{!(delivery.actionString=='reviewAssessment' || delivery.actionString=='gradeAssessment')}">
-<f:verbatim><br/></f:verbatim>
-<h:outputText value="#{deliveryMessages.maxSAText}"/>
+<br/>
+<h:panelGroup>
+        <h:outputText value='<br/>
+        <button type="button" id="getAaCharCount" class="active" onclick="clickSaCharCountOnFront(this);">#{deliveryMessages.count_characters}</button>
+        &#160;' escape='false' />
+        <h:inputText value="#{question.saCharCount}" size="8" readonly="true" style="text-align:right;"/>
```

Comment:
I'd prefer to see this without the `<br />`, use CSS for styling. Same with the inline styles on the `<h:inputText>`; styles should be isolated in CSS files. Also, I'm not sure having this `<button>` in the value of an `<h:outputText>` is the right approach.